documentation/onboarding.md

```diff
@@ -0,0 +1,169 @@
+
+![Image showing the User Account Control dialog highlighting the Yes button](./images/onboarding/user-account-control.png)
+
+The installation process begins. Allow a few minutes for the installation to complete.
+
+![Image showing the install status bar](./images/onboarding/installing.png)
+
+Agenda is successfully installed on the system and a shortcut icon is created on the desktop.
+
+![Image showing the desktop shortcut](./images/onboarding/desktop-shortcut.png)
+
+Please see the section on MacOS install for more information on how to download and install the desktop client to a Mac machine.
+
+## MacOS install
+
+The Agenda application is compatible with MacOS and can be installed on the MacOS Catalina and newer operating systems.
+
+>[!NOTE] 
+>
+>You can use any web browser to download the app.
+
+To download the Agenda desktop client on MacOS:
+
+Navigate to https://www.example.agenda.ca/download-app. This is the official Agenda download site.
+
+Select the **[!UICONTROL Download]** button for MacOS. 
+
+![Image showing the download site highlighting the MacOS download button](./images/onboarding/download-center-mac.png)
+
+The file is downoaded and saved to your machine.
+>[!NOTE]
+>
+>The default destination for downloads on a Mac is the **Downloads** folder.
+
+Navigate to the Downloads folder and select the downloaded installer.
+
+![Image showing the downloaded installer in the Downloads folder](./images/onboarding/launch-installer-mac.png)
+
+The installer dialog opens the [!UICONTROL Install] dialog on the **Introduction** tab. Select **[!UICONTROL Continue]**.
+
+![Image showing the installer dialog on the Introduction tab highlighting the Continue button](./images/onboarding/installer-intro.png)
+
+You are taken to the **Installation Type** tab where you can see how much space is required to install the application. Select **[!UICONTROL Install]**.
+
+![Image showing the installer dialog on the Installation Type tab highlighting the Install button](./images/onboarding/installer-installation-type.png)
+
+You are presented with the installer credentials dialog box. Enter the administrator credentials (Username and Password) for your device and select **[!UICONTROL Install Software]**.
+
+![Image showing the installer credentials dialog highlighting the Install Software button](./images/onboarding/installer-credentials.png)
+
+Allow a few minutes for the installation to complete. A dialog appears confirming the installation was successful. Select **[!UICONTROL Close]**.
+
+![Image showing the installer dialog on the Summary tab confirming installation was successful highlighting the Close button](./images/onboarding/installation-successful.png)
+
+The Agenda app is now available in the **Launchpad**.
+
+![Image showing the Agenda app in the Launchpad](./images/onboarding/launchpad.png)
+
+## Create an account
+
+An Agenda account gives you access to many Agenda products. With an Agenda account you can do the following:
```

Comment:
Have we established that Agenda has "many" products? Agenda is just the one singular product AFAIK, so I think you're actually talking about capabilities or features here.

Let's also make it simple and say an Agenda account gives you access to _all_ of its features, not just "many". Maybe at a later iteration we could work in different tiers or SKUs for Agenda into the docs, but for now we'll just say it's a one-size-fits-all kinda deal.